src/compositions/question-form-11.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react';
+import { InputField } from '../elements/input-field';
+import { useRouter } from 'next/dist/client/router';
+
+type Props = {
+  label?: string;
+  initialAnswer?: string;
+  href: string;
+  answer: { [key: string]: string };
+  callback: React.Dispatch<React.SetStateAction<{ [key: string]: string }>>;
+};
+
+const validateInput = (answer: string) => {
+  if (parseFloat(answer) > 10000) {
+    return 'Bist du sicher? das sind sehr viele km.';
+  } else if (answer.length > 10) {
+    return 'Bitte gib maximal 10 Ziffern nach dem Komma ein.';
+  }
+  return null;
+};
+
+export const Question11: FC<Props> = ({ label, initialAnswer, href, answer, callback }) => {
+  const router = useRouter();
+
+  return (
+    <div>
+      <InputField
+        type="number"
+        label={label || 'km'}
+        name="answer"
+        id="number"
+        step="10"
+        placeholder={initialAnswer || '100'}
+        value={answer?.km}
+        onChange={(value) => {
+          callback({ km: value });
+        }}
+        onKeyDown={(key) => key === 'Enter' && router.push(href)}
```

Comment:
Why does `Enter` always result in a page navigation but when doing it with a mouse I have to click "Weiter"? The `onKeyDown` doesn't seem very intuitive...